### DISCUSSION TITLE
Thermocline Depth Calculation

### DISCUSSION THREAD

**Discussion by Mohima17** [ProjectPythia/projectpythia.github.io#278] [Q&A]
Hello Project Pythia!

How can I easily calculate the depth of 20 degree celsius temperature with the xarray library?

Also, the depth where the sound velocity is maximum (sonic layer depth).

Thanks in advance.

**Comment by clyne** [comment]
Hi @Mohima17,

Apologies that your question fell through the cracks somehow. This probably happened because we don't have an answer for you :-(. You might try posting your question on the Pangeo [forum](https://discourse.pangeo.io/). Good luck to you!
